makeGridPacks.py

```diff
@@ -104,8 +99,24 @@
 
 import os
 import subprocess
-f=file('makeGridPacks.mg','w')
+
 if __name__=='__main__':
+    # Parse arguments
+    from sys import argv,exit
+
+    if len(argv)<3:
+        print('usage: {} E process [test]'.format(argv[0]))
+        sys.exit(1)
+
+    E=int(argv[1])
+    process=argv[2]
+    if len(argv)>3: test=bool(int(argv[3]))
+
+    # Currnt working directory for absolute path for resources
+    prodBase=os.path.dirname(os.path.realpath(__file__))
```

Comment:
is this safer than using the environment variable?  Or do you just want to be able to execute this script from any dir?  I just want to understand the motivation

Reply:
I'm not a big fan of having to run a setup script at the start of every session. This automates the procedure of finding everything.

I've updated the code to optionally use `prodBase` env variable, falling back to the location of the script only when undefined. This should be compatible with everyone's workflow.

Reply:
That makes sense.  I was just worried that `lepMult/test.sh` and `generateGridpacks.sh` use the environment variable and was worried about scripts getting out of sync.  But this is a perfect solution.  Thanks!